### DISCUSSION TITLE
請問可不可以在中文的模式下輸入半角標點符號呢?

### DISCUSSION THREAD

**Discussion by raychunghk** [rime/home#1193] [Q&A]
請問可不可以在中文的模式下輸入半角標點符號呢?

**Comment by pdog18** [comment]
可以通过「[定制](https://github.com/rime/home/wiki/CustomizationGuide#%E4%B8%80%E4%BE%8B%E5%AE%9A%E8%A3%BD%E6%A8%99%E9%BB%9E%E7%AC%A6%E8%99%9F)」 实现

**Comment by gshmu** [comment]
正解應該是：Ctrl+` 呼出菜單，然後選擇半角符號。

默認使用半角符號，可以通過改配置文件實現
